packages/keycodes/src/index.js

```diff
@@ -372,6 +372,12 @@ export const isKeyboardEvent = mapValues( modifiers, ( getModifiers ) => {
 			key = String.fromCharCode( event.keyCode ).toLowerCase();
 		}
 
+		// Replace some characters to match the key indicated
+		// by the shortcut when a modifier key is pressed.
+		if ( event.shiftKey && character.length === 1 ) {
+			key = key.replace( '<', ',' );
```

Comment:
Thanks for working on this. Is it a recent regression? It's surprising to me as this code seems to have been mostly unchanged for a while, and I would've expected quite a few bug reports.

I don't think this is the right solution as it makes an assumption that the user's keyboard has a layout where <kbd><</kbd> and <kbd>,</kbd> share a key.

An example of where this wouldn't work is that many European keyboard layouts (Danish, Dutch, Italian) have <kbd>;</kbd> and <kbd>,</kbd> sharing a key (https://en.wikipedia.org/wiki/List_of_QWERTY_keyboard_language_variants).

I think it'd be better to try to understand from the event payload which key was pressed (e.g. it looks like `event.code === 'Comma'`)

Reply:
@talldan It is a bug for sure. For example, pressing ctrl+` will move through landmarks but ctrl+shift+` no longer works. It has been this way for some time so I was hoping this PR could at least show us what was going wrong for more fixes.

I cannot remember the last time any of these worked well on Windows...